tests/data_context/cloud_data_context/test_checkpoint_crud.py

```diff
@@ -118,53 +117,53 @@ def _mocked_get_response(*args, **kwargs):
+                                "self": f"/organizations/{organization_id}/checkpoints/{checkpoint_id}/relationships/attributes",
                             }
                         },
                         "clauses": {
                             "links": {
-                                "related": f"/organizations/{organization_id}/contracts/{contract_id}/clauses",
-                                "self": f"/organizations/{organization_id}/contracts/{contract_id}/relationships/clauses",
+                                "related": f"/organizations/{organization_id}/checkpoints/{checkpoint_id}/clauses",
+                                "self": f"/organizations/{organization_id}/checkpoints/{checkpoint_id}/relationships/clauses",
                             }
                         },
                         "created_by": {
                             "links": {
-                                "related": f"/organizations/{organization_id}/contracts/{contract_id}/created-by",
-                                "self": f"/organizations/{organization_id}/contracts/{contract_id}/relationships/created-by",
+                                "related": f"/organizations/{organization_id}/checkpoints/{checkpoint_id}/created-by",
+                                "self": f"/organizations/{organization_id}/checkpoints/{checkpoint_id}/relationships/created-by",
                             }
                         },
                         "organization": {
                             "links": {
-                                "related": f"/organizations/{organization_id}/contracts/{contract_id}/organization",
-                                "self": f"/organizations/{organization_id}/contracts/{contract_id}/relationships/organization",
+                                "related": f"/organizations/{organization_id}/checkpoints/{checkpoint_id}/organization",
+                                "self": f"/organizations/{organization_id}/checkpoints/{checkpoint_id}/relationships/organization",
                             }
                         },
                         "suite_validation_results": {
                             "links": {
-                                "related": f"/organizations/{organization_id}/contract/{contract_id}/suite-validation-results",
-                                "self": f"/organizations/{organization_id}/contracts/{contract_id}/relationships/suite-validation-results",
+                                "related": f"/organizations/{organization_id}/checkpoint/{checkpoint_id}/suite-validation-results",
+                                "self": f"/organizations/{organization_id}/checkpoints/{checkpoint_id}/relationships/suite-validation-results",
                             }
                         },
                     },
-                    "type": "contract",
+                    "type": "checkpoint",
                 },
                 "jsonapi": {"version": "1.0"},
                 "links": {
```

Comment:
note: I don't think you're using these, but we're going to drop `links` in all our future APIs